[debug] [2019-06-21T22:10:56.780Z] ----------------------------------------------------------------------
[debug] [2019-06-21T22:10:56.782Z] Command:       /usr/local/bin/node /usr/local/bin/firebase open
[debug] [2019-06-21T22:10:56.782Z] CLI Version:   7.0.0
[debug] [2019-06-21T22:10:56.782Z] Platform:      darwin
[debug] [2019-06-21T22:10:56.782Z] Node Version:  v8.12.0
[debug] [2019-06-21T22:10:56.782Z] Time:          Fri Jun 21 2019 15:10:56 GMT-0700 (PDT)
[debug] [2019-06-21T22:10:56.782Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-06-21T22:10:56.789Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-06-21T22:10:56.789Z] > authorizing via signed-in user
[debug] [2019-06-21T22:10:56.789Z] [iam] checking project tt-angular-858f9 for permissions ["firebase.projects.get"]
[debug] [2019-06-21T22:10:56.790Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/tt-angular-858f9:testIamPermissions  

[debug] [2019-06-21T22:10:56.936Z] <<< HTTP RESPONSE 200
[debug] [2019-06-21T22:10:56.938Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/tt-angular-858f9  
 
[debug] [2019-06-21T22:10:57.072Z] <<< HTTP RESPONSE 200
[debug] [2019-06-21T22:10:57.073Z] TypeError: prompt.once is not a function
    at Command._action (/usr/local/lib/node_modules/firebase-tools/lib/commands/open.js:69:14)
    at /usr/local/lib/node_modules/firebase-tools/lib/command.js:163:37
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:229:7)
[error] 
[error] Error: An unexpected error has occurred.
